### DISCUSSION TITLE
Migrate from GitLab

### DISCUSSION THREAD

**Discussion by yan-ad** [kiriminaja/php#13] [Announcements]
FYI Guys Mulai dari 11 Januari 2022 dari lead memutuskan untuk melakukan pemindahan git management dari gitlab dengan alasan sebagai berikut

- Ekosistem Github lebih proper untuk mendukung kebutuhan kerja
- Dukungan komunitas lebih besar
- Optimasi pembiayaan tim

**Comment by gemaantikahr** [comment]
testing

**Comment by yan-ad** [comment]
balas

**Reply by yan-ad** [reply]
Test Bot Discord @kiriminaja/kiriminaja-backend

**Reply by yan-ad** [reply]
Mantap iso guys
![apple-badge](https://user-images.githubusercontent.com/39618526/211764452-fd048384-38f0-49d0-8149-55679b2d60cd.png)

**Comment by yan-ad** [comment]
KAJ-2613

**Comment by yan-ad** [comment]
[KAJ-2613]
